src/components/qf/components/QFProjectCard.tsx

```
@@ -40,13 +40,19 @@ export function QFProjectCard({
   const isInCart = checkIsInCart(projectId, roundId)
   const numericProjectId = Number(project.id)
 
-  const formatCurrency = (value: number) =>
-    new Intl.NumberFormat('en-US', {
-      style: 'currency',
-      currency: 'USD',
-      maximumFractionDigits: 0,
-    }).format(value)
+ const formatCurrency = (value: number) => {
```

ESLint

[error] Insert `·`

(prettier/prettier)